ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml

```diff
@@ -143,6 +143,24 @@ tests:
         OPENSHIFT_INSTALL_RELEASE_IMAGE=${RELEASE_IMAGE_LATEST}
         OPENSHIFT_VERSION=4.8
     workflow: baremetalds-assisted-install
+- as: e2e-metal-assisted-kube-api
+  steps:
+    cluster_profile: packet
+    dependencies:
+      ASSISTED_SERVICE_IMAGE: pipeline:assisted-service
+    env:
+      ASSISTED_CONFIG: |
+        ENABLE_KUBE_API=true
+        NUM_MASTERS=1
+        NUM_WORKERS=0
+        IPv6=yes
+        IPv4=no
+        PROXY=yes
+        ISO_IMAGE_TYPE=minimal-iso
+        KUBE_API=yes
```

Comment:
Why there are both `ENABLE_KUBE_API` and `KUBE_API`?

Reply:
This naming is really unclear...

Reply:
Yes, we will need to discuss some renamings